### ISSUE TITLE
Duplicated bootstrap components causing trouble

### ISSUE THREAD

**Issue by alxndrsn** [enketo/enketo-core#302]
It looks like `select-desktop/selectpicker.js` widget contains the bootstrap-dropdown plugin.  This appears to conflict with our bootstrap-dropdown from `dropdown.js` in bootstrap's core.

Am I doing something wrong with the widget configuration?  If not, could enketo include bootstrap sources via `require()` calls, allowing override with aliases in our configuration?  And if this is not possible, would it be suitable to namespace enketo's `jQuery.fn.dropdown` (I assume it's this that's causing the problem).

**Comment by MartijnR** [comment]
That conflict is possible... After getting rid of bootstrap I took a shortcut and copied any remain bootstrap dependencies into the relevant widget(s).

An easy solution would perhaps be to use the old widget (which is still present in enketo-core) called select-desktop-bootstrap. I haven't updated that widget for a while, so maybe one or two fixes in select-desktop would have to be copied to select-desktop-bootstrap. There were a few critical fixes in the past few months (probably these were css-only, not sure). 

If this is not a solution, I would be happy if that particular bootstrap plugin was included with require() and if Enketo's own jQuery.fn.dropdown was removed (until we have a cleaner lib-agnostic select widget).
